command/results_xunit.go

```diff
@@ -162,3 +152,39 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 
 	return sendJSONResults(ctx, conf, logger, comm, &task.TestResults{tests})
 }
+
+func generateLogsForOneFile(testSuites []testSuite, t *task.Task, tests []task.TestResult, logs []*model.TestLog, logIdxToTestIdx []int) ([]task.TestResult, []*model.TestLog, []int) {
+	// go through all the tests
+	for _, suite := range testSuites {
+		for _, tc := range suite.TestCases {
+			// logs are only created when a test case does not succeed
+			test, log := tc.toModelTestResultAndLog(t)
+			if log != nil {
+				logs = append(logs, log)
+				logIdxToTestIdx = append(logIdxToTestIdx, len(tests))
+			}
+			tests = append(tests, test)
+		}
+
+		if suite.SysOut != "" {
+			log := model.TestLog{
+				Name:          suite.Name,
+				Task:          t.Id,
+				TaskExecution: t.Execution,
+				Lines:         []string{fmt.Sprintf("system-out: %s", suite.SysOut)},
```

Comment:
how do we distinguish between stdout/stderrr otherwise?

Why not just "system-out" on one line, and then have the other on the next, to avoid the sprintf

Reply:
I was thinking: 

`Lines: []string{"system-out", suite.SysOut},`

but seems totally optional either way.